<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1394</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DiagramPlot" name="bifurPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1041</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>550</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcBtn">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>610</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Animate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="angleEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>45</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle (grad)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V left</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vLeftEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V right</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vRightEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redrawBtn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>550</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Redraw</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>grid size X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gridSizeXEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>deviation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deviationEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>210</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>dt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dtEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1e-7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chargeEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-1.6e5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>charge</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="massEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>210</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9.11e-7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mass</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gridSizeYEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>grid size Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="activeGridSizeEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>active grid size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="particlesAmountEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>330</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>310</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Particles amount</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="ivPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>1041</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="plotIVBtn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>610</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot IV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>310</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>uStepsCount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uStepsCountEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>330</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>40</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uMaxEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>390</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>15</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>370</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>uMax</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uMinEdit">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>390</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>370</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>uMin</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DiagramPlot</class>
   <extends>QWidget</extends>
   <header location="global">diagramplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
